libipl/p10/ipl0.C

```diff
@@ -755,6 +755,9 @@ static int ipl_set_ref_clock(void)
 		if (!ipl_is_functional(proc))
 			continue;
 
+		if (!ipl_is_master_proc(proc))
```

Comment:
Need to restructure the code to support hwp execution/error handling only on primary processor.

Refer sbe_config_update function.